pyfolio/tests/test_tears.py

```diff
@@ -26,7 +26,7 @@ class PositionsTestCase(TestCase):
     test_returns = read_csv(
         gzip.open(
             __location__ + '/test_data/test_returns.csv.gz'),
-        index_col=0, parse_dates=True)
+        index_col=0, parse_dates=True).iloc[4:]
```

Comment:
Why do we skip the first 4 rows for the returns CSV, but don't skip rows for the positions or transactions CSVs?